crates/polars-stream/src/async_primitives/mod.rs

```diff
@@ -1,5 +1,7 @@
 pub mod connector;
 pub mod distributor_channel;
 pub mod linearizer;
+#[allow(unused)]
```

Comment:
`expect(unused)` ensures you don't forget to remove it once it's used :) Rest lgtm.